### DISCUSSION TITLE
To wide gap fill in model

### DISCUSSION THREAD

**Discussion by alexandrkut** [supermerill/SuperSlicer#1095] [Q&A]
Hello.
In my model i have to wide gap fill.
Nozzle 0,6, but gap fill try make 1.2mm line (and cannot make it).
![image](https://user-images.githubusercontent.com/32929279/115122587-53e5b680-9fc1-11eb-942c-7f5f9fb3596d.png)

![image](https://user-images.githubusercontent.com/32929279/115122595-619b3c00-9fc1-11eb-91a6-d8e03c0262e9.png)

In prusa slicer i turn off thin wall and all ok (in super slicer this not help).
![image](https://user-images.githubusercontent.com/32929279/115122630-8db6bd00-9fc1-11eb-9cea-48ccf2ac24d8.png)

How i can turn off wide gap fill?
Thank you!

Model:

[beta.zip](https://github.com/supermerill/SuperSlicer/files/6329778/beta.zip)

**Comment by neophyl** [comment]
generally as we are dealing with settings the model on its own is not very useful.  If you save your Project (File>Save Project as) to save the .3mf and then attach that we can then see the same as you as all your settings will be in the project.

**Reply by alexandrkut** [reply]
3MF in attach
[gap_fill.zip](https://github.com/supermerill/SuperSlicer/files/6404431/gap_fill.zip)

**Comment by supermerill** [comment]
activate overlapping perimeters for all

**Reply by alexandrkut** [reply]
Thank you. it's work!

**Reply by Funkton** [reply]
That fixed it for me too. I'd highly recommend changing the default for that setting. All of my issues with Super Slicer were due to this. I was wondering why I had 20+% gap fills and overextruded blobs in the middle of walls.

**Reply by supermerill** [reply]
Activating overlapping perimeters can create massive overextrusion, i won't activate it by default.
If you had a huge problem with a big fill, feel free to submit an issue.
